src/sections/map/maps-component.tsx

```diff
@@ -28,6 +28,17 @@ export default function MapComponent({
   const mapRef = useRef<HTMLDivElement>(null);
   const leafletMapRef = useRef<L.Map | null>(null);
   const markerGroupRef = useRef<L.LayerGroup | null>(null);
+  const [currentTime, setCurrentTime] = useState<string>("");
+
+  useEffect(() => {
+    const updateTime = () =>
+      setCurrentTime(
+        new Date().toLocaleString("en-KE", { timeZone: "Africa/Nairobi" })
+      );
+    updateTime();
+    const interval = setInterval(updateTime, 60_000);
+    return () => clearInterval(interval);
+  }, []);
```

Comment:
Why are we storing current time in state? I don't think we need to do that for current time.